### PR TITLE
try cleanup some more before exit

### PR DESCRIPTION
**Pull request by adizaimi**

### CODE REVIEW

File: bb/object_pool.c [hunk 1]
```diff
@@ -312,6 +310,7 @@ int comdb2_objpool_destroy(comdb2_objpool_t op)
 
     /* clear access history */
     hash_for(op->history, hash_elem_free_wrapper, NULL);
+    hash_clear(op->history);
```
Comment (riverszhang89) [inline bb/object_pool.c]: You don't need to call `hash_clear()` as `hash_free()` clears entries as well.

Comment (adizaimi) [reply bb/object_pool.c]: good point, removing, thanks.

**Comment by riverszhang89** [comment]
Wait.. You didn't remove the redundant code I pointed out..

**Comment by adizaimi** [comment]
Doing it in https://github.com/bloomberg/comdb2/pull/620 -- changed in local branch but forgot to push. 
Thanks.

**Pull request merged**